### DISCUSSION TITLE
will osu lazer scores be erased when laser replaces stable or whatever

### DISCUSSION THREAD

**Discussion by eatcakeisstupid** [ppy/osu#27089] [Q&A] [answered]
I never farmed much and play mcosu because stable runs like trash on mac... ive already farmed a lot and should've thought about this first

**Comment by peppy** [comment, marked as answer]
scores set since the start of february will most probably not be deleted. previous scores set on lazer from 2021-2023 will/have been, though.

**Reply by eatcakeisstupid** [reply]
oh ok

I mainly asked because [this play](https://media.discordapp.net/attachments/855691437886013480/1204972377151246376/OKGMGOGmogOmGOOGOMGMOGMOGOMGMOGMOL.png?ex=65d6acfa&is=65c437fa&hm=d5120cb9e60f2b1f4e3619f3d3f30a2e700029a0fdb7af783fbd252b37ce9e6b&=&format=webp&quality=lossless&width=1814&height=1020) but I just noticed I don't have the 8 star fc badge on my profile? https://osu.ppy.sh/users/12485342 so will badges come soon or preferably be given from lazer plays soon? no way I'm doing that again :(

**Reply by Stoppedpuma** [reply]
Badges aren't here quite yet but will be soon.
